DSCResources/MSFT_xWebsite/MSFT_xWebsite.schema.mof

```diff
@@ -1,27 +1,28 @@
-[ClassVersion("1.0.0")]
-Class MSFT_xWebBindingInformation
+[ClassVersion("1.1.0")]
```

Comment:
Please do not updating the Class version, if this is needed it will be done during the release process.  I have requested that the [contribution guidelines](https://github.com/PowerShell/DscResources/blob/TravisEz13-patch-1/CONTRIBUTING.md#creating-a-new-dsc-resource-in-an-existing-module) be updated.  Currently, only the module version should need to be updated for PowerShell DSC Resources (which will also be updated during the release process per the existing contribution guideline.)